c/ldv-memsafety-bitfields/test-bitfields-2.1_false-valid-free.c

```diff
@@ -7,8 +7,10 @@ struct A {
 	unsigned char a;
 	unsigned char b:2;
 	unsigned char c:2;
+	unsigned char pad1[2];
```

Comment:
why is the padding needed?
accessing p->d or p->e should not result in errors with and without packing.